addons/binding/org.openhab.binding.meteoblue/README.md

```diff
@@ -123,19 +123,18 @@ Group weatherDay6 "Weather in 6 days"
 
 // ----------------- meteoblue ITEMS -------------------------------------------
 DateTime todayForecastDate  "Forecast for [%1$tY/%1$tm/%1$td]"  <calendar>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#forecastDate"}
-String todayPCode    "Pictocode [%d]"  <iday>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#condition"}
-String todayCond     "Condition [%s]"  <iday>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#condition"}
-String todayIcon     "Icon [%s]"       (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#icon"}
+String todayCond     "Condition [%s]"  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#condition"}
+Image todayIcon     "Icon"       (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#icon"}
 Number todayUV       "UV Index [%d]"  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#UVIndex"}
-Number:Temperature  todayTempL  "Low Temp [%.2f �F]"   <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#minTemperature"}
-Number:Temperature  todayTempH  "High Temp [%.2f �F]"  <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#maxTemperature"}
+Number:Temperature  todayTempL  "Low Temp [%.2f °F]"   <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#minTemperature"}
+Number:Temperature  todayTempH  "High Temp [%.2f °F]"  <temperature>  (weatherDay0)  {channel="meteoblue:weather:metBridge:A51:forecastToday#maxTemperature"}
```

Comment:
Please remove these Meteoblue changes from the PR.

Reply:
I'm a newbie of github, I don't know why this commit include in my PR. I don't know how to remove it from my PR.  Can you help me learning how to do that.

Reply:
Just add a new commit that modifies the file so the file has the content of the current master again:

https://github.com/openhab/openhab2-addons/blob/master/bundles/org.openhab.binding.meteoblue/README.md